src/platform/datapath_raw_xdp.c

```diff
@@ -56,10 +60,17 @@ typedef struct _XDP_QUEUE {
     CXPLAT_LIST_ENTRY TxQueue;
 } XDP_QUEUE;
 
+typedef struct XDP_FLOW {
+    const XDP_INTERFACE* Interface;
+    CXPLAT_LIST_ENTRY Link;
+    XDP_QUIC_FLOW Descriptor;
+    XDP_QUEUE Queues[0];
```

Comment:
what it this? It's not defined anywhere.

Reply:
Btw, it doesn't seem to compile.

Reply:
Yes it does: https://dev.azure.com/ms/msquic/_build/results?buildId=249521&view=results

Reply:
I can't find the definition of XDP_QUIC_FLOW. Where is it?

Reply:
It's in the latest XDP header (program.h I beleive). You need to update your XDP to have it.

Reply:
ah I see.